QualityControl/docs/Installation.md

```diff
@@ -0,0 +1,74 @@
+# Installation
+
+Follow these steps to install and set up QCG (Web UI for O<sup>2</sup> Quality Control).
+
+## 1. Install Node.js (version 22.x or higher)
+
+To run the application, you'll need Node.js version `>= 22.x`. Choose your operating system below to install Node.js:
+
+- **On CentOS 7 (CC7):**
+This command installs the correct version of Node.js via the NodeSource repository.
+    ```bash
+    yum install https://rpm.nodesource.com/pub_16.x/el/8/x86_64/nodejs-16.9.1-1nodesource.x86_64.rpm
+    ```
+
+
+- **On macOS (using Homebrew):**
```

Comment:
I believe for MacOS, the installation recommendation changed: https://nodejs.org/en/download
They recommend using nvm over brew